gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -63,26 +63,29 @@
   private final CallOptions callOptions;
   @Nullable private final Duration streamWaitTimeout;
   @Nullable private final Duration streamIdleTimeout;
+  @Nullable private final Integer channelAffinity;
```

Comment:
Please strongly consider making affinity of type `Long/long` (here and everywhere else).